Apps/Admin/Client/Pages/DashboardPage.razor

```diff
@@ -123,7 +123,7 @@
                 <MudChart
                     data-testid="year-of-birth-chart"
                     ChartType="ChartType.Bar"
-                    ChartSeries="@(new List<ChartSeries> { GetYearOfBirthCountSeries() })"
+                    ChartSeries="[GetYearOfBirthCountSeries()]"
```

Comment:
Sorry don't have time to test this, but is it correct to drop the @ to ensure its executed as a C# expression?

if its fine please just close this.

Reply:
I did see this in another template as well so I don't think my concerns are really an issue.

Reply:
The only reason I dropped the @ was because the formatter was being weird and adding an extra space character at the start of the string when it was there.  I did test it out to be sure it would still work.